static_src/components/user_provider.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
_please excuse my non-react knowledge_. what's the purpose of this file? i see it keeps the state of the currentUser but couldn't we just have the component get the currentUser from the store directly?